docs/content/en/docs/yaml-crd-ref/config.md

```diff
@@ -27,46 +27,34 @@ spec:
 
 * **metadata**
   * **name** -- Unique name of this set of configurations.
-    * Must be an alphanumeric string and, by convention, is all lowercase.
-    * Can include the special characters `_`, `-`, (others?)
-    * Should not include spaces.
+    Names must comply with the
+    [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
+    specification.
 
 * **spec**
   * **oTelCollectorUrl** -- The URL and port of the OpenTelemtry collector
   * **keptnAppCreationRequestTimeoutSeconds** --
-     interval in which automatic app discovery searches for workloads
-     to put into the same auto-generated [KeptnApp](app.md).
+    interval in which automatic app discovery searches for workloads
+    to put into the same auto-generated [KeptnApp](app.md).
 
-    Both values can be defined in one CRD
-    or you can create separate `KeptnApp` CRDs for each one.
+Each [KeptnApp](app.md) should have a single `KeptnConfig` CRD
+that describes all configurations for that application.
```

Comment:
That's not actually the case. You can have one `KeptnConfig` per KLT. So usually users should only have one config in their cluster.

Reply:
done